xep-0353.xml

```diff
@@ -237,48 +431,123 @@
+  <p>
+    It is possible that a &lt;propose/&gt; message can be sent at the same time
+    by both parties or a new session started while one is already running. In
+    case of two-party calls (when the 'multi' attribute of both &lt;propose&gt;
+    element is not set to 'true'), implementations of this specification MUST
+    implement the following solutions to solve this. (This is loosely based upon
+    section 7.2.16 of &xep0166;.)
+  </p>
   <section2 topic='No existing session' anchor='tie-break-1'>
-    <p>In this case (e.g. no party answered the &lt;propose/&gt; message yet) the lower of the two session IDs MUST overrule the other action, where by "lower" is meant the session ID that is sorted first using "i;octet" collation as specified in Section 9.3 of &rfc4790; (in the unlikely event that the random session IDs are the same, the action sent by the lower of the JabberIDs MUST overrule the other action). The party that receives the &lt;propose/&gt; action with the lower of the two session IDs MUST respond with an &lt;accept/&gt; or &lt;reject/&gt; mesage like it would normally do for a &lt;propose/&gt; message, and the party that receives the &lt;propose/&gt; action with the higher of the two session IDs MUST return a &lt;reject/&gt; message to the other party with a &lt;tie-break/&gt; child element alongside of a &lt;reason/&gt; element carrying the condition &lt;expired/&gt;.</p>
+    <p>
+      In this case (e.g. no party answered the &lt;propose/&gt; message yet)
+      the lower of the two session IDs MUST overrule the other action, where by
+      "lower" is meant the session ID that is sorted first using "i;octet"
+      collation as specified in Section 9.3 of &rfc4790; (in the unlikely event
+      that the random session IDs are the same, the action sent by the lower of
+      the JabberIDs MUST overrule the other action). The party that receives the
+      &lt;propose/&gt; action with the lower of the two session IDs MUST respond
+      with an &lt;accept/&gt; or &lt;reject/&gt; message like it would normally
+      do for a &lt;propose/&gt; message, and the party that receives the &lt;propose/&gt;
+      action with the higher of the two session IDs MUST return a &lt;reject/&gt;
+      message to the other party with a &lt;migrated/&gt; child element whose
+      to-attribute is set to the lower session ID and a &lt;reason/&gt; element
+      carrying the condition &lt;tie-break/&gt;.
+    </p>
       <example caption="Tie break in propose state"><![CDATA[
 <!-- lower session ID -->
 <message from='romeo@montague.example/orchard'
          to='juliet@capulet.example'
          type='chat'>
-  <propose xmlns='urn:xmpp:jingle-message:1' id='a73sjjvkla37jfea'>
-    <description xmlns='urn:xmpp:jingle:apps:rtp:1' media='audio'/>
```

Comment:
the intent of putting the media into was to allow a phone to reject a video call when receiving a push message. That use-case remains valid.

Reply:
The information of the media being used in the call is still available via the `audio` and `video` attributes on the `<propose>` (audio defaulting to true, video defaulting to false, as described in the section "Invite to a Call").

Also, the clients can still negotiate to not accept the video content and only accept the audio content when doing the Jingle connection establishment.